src/main/java/net/syscon/elite/api/model/OffenderSentenceDetail.java

```diff
@@ -17,7 +17,7 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @AllArgsConstructor
 @NoArgsConstructor
-@EqualsAndHashCode(callSuper = true)
+@EqualsAndHashCode(callSuper = false)
```

Comment:
not sure about this change - won't it then mean two sentence details can be equal for different offenders?

Reply:
I missed the Data annotation and assumed that there wasn't an equals/hashcode on the super class
I've removed this change as it wasn't needed any more.